backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -376,9 +387,7 @@ func (h *Handler) playerSelectTopic() {
 }
 
 type PlayTimeSend struct {
-	LockCanvas bool `json:"lockCanvas,omitempty"`
-	LockChat   bool `json:"lockChat,omitempty"`
-	Score      int  `json:"score,omitempty"`
+	Score int `json:"score,omitempty"`
```

Comment:
What is omitempty?

Reply:
If the field is empty, it'll omit the field in the json